husky_bringup/launch/ft_300/ft_300_right.launch

```diff
@@ -0,0 +1,15 @@
+<launch>
```

Comment:
For the same reason above - parametrization cannot be made here. merge is possible.